oshdb-api/src/test/java/org/heigit/ohsome/oshdb/api/tests/TestOSHDBFilter.java

```diff
@@ -33,21 +35,34 @@
    * Creates a test runner using the H2 backend.
    * @throws Exception if something goes wrong.
    */
-  public TestOhsomeFilter() throws Exception {
+  public TestOSHDBFilter() throws Exception {
     OSHDBH2 oshdb = new OSHDBH2("./src/test/resources/test-data");
     filterParser = new FilterParser(new TagTranslator(oshdb.getConnection()));
     this.oshdb = oshdb;
   }
 
-  private MapReducer<OSMEntitySnapshot> createMapReducer() throws Exception {
+  private MapReducer<OSMEntitySnapshot> createMapReducerOSMEntitySnapshot() throws Exception {
     return OSMEntitySnapshotView.on(oshdb)
         .areaOfInterest(bbox)
         .timestamps("2014-01-01");
   }
 
+  private MapReducer<OSMContribution> createMapReducerOSMContribution() throws Exception {
+    return OSMContributionView.on(oshdb)
+        .areaOfInterest(bbox)
+        .timestamps("2008-01-01", "2014-01-01");
```

Comment:
With this time range, I suppose the idea is that `createMapReducerOSMContribution()` contains the same result as `createMapReducerOSMEntitySnapshot()`. I think that's okay for now since the contribution endpoint is tested properly in other oshdb-api tests, correct?

Reply:
yes, that was the idea. since there are no matching-but-later-deleted elements in this test data set, the results should match.